### PR TITLE
how-to: Improve large image pull documentation

### PR DESCRIPTION
**Pull request by JakubLedworowski**

### CODE REVIEW

File: docs/how-to/how-to-pull-images-in-guest-with-kata.md [hunk 1]
```diff
@@ -178,135 +178,217 @@ Currently, the image pulled in the guest will be downloaded and unpacked in the
...
+          persistentVolumeClaim:
+            claimName: trusted-pvc
+      containers:
+        - name: app-container
+          image: quay.io/confidential-containers/test-images:largeimage
+          command: ["/bin/sh", "-c"]
+          args:
+            - sleep 6000
+          volumeDevices:
+            - devicePath: /dev/trusted_store
```
Comment (Apokleos) [inline docs/how-to/how-to-pull-images-in-guest-with-kata.md:323]: Is`/dev/trusted_store` a pre-defined device path here in CoCo ? Some comments for this path might be helpful for users to understand what and why.

File: docs/how-to/how-to-pull-images-in-guest-with-kata.md [hunk 2]
```diff
@@ -178,135 +178,217 @@ Currently, the image pulled in the guest will be downloaded and unpacked in the
...
+    ```
+
+    See `overlay` of size `10G` mounted on `/`:
+    ```text
+    Filesystem                Size      Used Available Use% Mounted on
+    overlay                   9.7G      2.0G      7.2G  22% /
+    tmpfs                    64.0M         0     64.0M   0% /dev
+    tmpfs                   927.8M         0    927.8M   0% /sys/fs/cgroup
+    (...)
+    ```
```
Comment (Apokleos) [inline docs/how-to/how-to-pull-images-in-guest-with-kata.md:346]:
```suggestion
```bash
$ df -h
  Filesystem                Size      Used Available Use% Mounted on
  overlay                   9.7G      2.0G      7.2G  22% /
  tmpfs                    64.0M         0     64.0M   0% /dev
  tmpfs                   927.8M         0    927.8M   0% /sys/fs/cgroup
  (...)
```
See `overlay` of size `10G` mounted on `/`
```

**Review by Apokleos — COMMENTED** [review]
Thx @JakubLedworowski I've been going through the changed code from start to finish, trying to understand how it works. But I'm still not sure about several things about it, so  comments left.

File: docs/how-to/how-to-pull-images-in-guest-with-kata.md (hunk 2 above)
Comment (JakubLedworowski) [reply docs/how-to/how-to-pull-images-in-guest-with-kata.md]: @Apokleos I did the split of command and output on purpose. I find it hard to follow all commands that are to be invoked because I need to copy each line manually from the code block. It is easier to copy the command directly using the `Copy` button and compare it to output in the next block.

File: docs/how-to/how-to-pull-images-in-guest-with-kata.md [hunk 3]
```diff
@@ -178,135 +178,217 @@ Currently, the image pulled in the guest will be downloaded and unpacked in the
...
+11. Create a file `trusted-storage-pod.yaml` with pulling large image in guest
+
+    ```yaml
+    ---
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: large-image-pod
+    spec:
+      runtimeClassName: kata-qemu
```
[the remark this answers is not in the document]
Comment (JakubLedworowski) [reply docs/how-to/how-to-pull-images-in-guest-with-kata.md]: You are right, I changed it to `kata-qemu-coco-dev` as this instruction is for confidential guest.

File: docs/how-to/how-to-pull-images-in-guest-with-kata.md (hunk 1 above)
Comment (JakubLedworowski) [reply docs/how-to/how-to-pull-images-in-guest-with-kata.md]: To be honest, I don't know how kata is matching the device for image pull. Tried to find it in the kata documentation but I failed. If any other reviewer can jump in here to explain I'll be glad to add this information to the PR.

**Comment by JakubLedworowski** [comment]
Hi @Apokleos can you please check the docs again? I improved the structure and addressed some of your comments. Thanks.

**Comment by JakubLedworowski** [comment]
Hi @fidencio, I'd be glad if you could jump in at your earliest convenience and review the changes. Thanks!
